torch/csrc/autograd/init.cpp

```diff
@@ -5,8 +5,12 @@
 #include <torch/csrc/autograd/grad_mode.h>
 #include <ATen/autocast_mode.h>
 #include <torch/csrc/autograd/profiler.h>
+#include <torch/csrc/autograd/record_function_ops.h>
```

Comment:
hmm, includes were not in order prior to this PR. Shall add one PR before this one to sort the includes in this file?

Reply:
https://github.com/pytorch/pytorch/pull/36401